packages/harmony/src/components/button/FollowButton/FollowButton.tsx

```diff
@@ -55,66 +60,81 @@ export const FollowButton = (props: FollowButtonProps) => {
     setIsHovering(false)
   }, [setIsHovering])
 
-  const handleChange = useCallback(() => {
+  const handleValueChange = useCallback(() => {
     if (value) {
       onUnfollow?.()
     } else {
       onFollow?.()
     }
     setValueState(!value)
-  }, [value, setValueState, onFollow, onUnfollow])
+  }, [value, setValueState, onUnfollow, onFollow])
+
+  const handleClick = () => {
```

Comment:
handleClick = handleValueChange cause otherwise that callback above doesnt matter